Guideline.md

```diff
@@ -282,6 +281,12 @@ At the event, make sure there are an appropriate number of sober staff on hand t
 #### 9.1 Name badges
 The purpose of a name badge is to identify people easily. As such, it should be easy to read! You should use a large, easy to read font. Sans serif fonts are generally regarded as a good choice. Make the name stand out and contrast well - avoid grey on grey! A font size of 18 points or bigger if you can make it fit would make a difference.
 
+Also consider allowing people to define or even write in their prefered name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send via email to allow people to identify themselves. Furthermore allow space for people to define their preferred pronouns or [provide premade buttons/stickers](http://yupgup.com/pronouns.html).
+
+Additionally it may make sense to provide stickers for people to denote things like "first time attendee", "ask me about [ - space to write in- ]", "talk to me" to forster more interaction between attendees.
```

Comment:
This is the what but not the why. It would be nice if you could add why it may make sense. 

Also a typo, forster -> foster

Reply:
the document in general doesn't go into all that much detail about everything and instead provides links for further reading .. in this spirit imho "foster more interaction between attendees" is sufficient, but if you have a link it would be good to add it.